src/blob_file_set.cc

```diff
@@ -284,6 +284,34 @@ Status BlobFileSet::MaybeDestroyColumnFamily(uint32_t cf_id) {
   return Status::NotFound("invalid column family");
 }
 
+Status BlobFileSet::DeleteBlobFilesInRanges(const RangePtr* ranges, size_t n,
+                                            bool include_end,
+                                            SequenceNumber obsolete_sequence) {
+  auto it = column_families_.find(cf_id);
+  if (it != column_families_.end()) {
+    VersionEdit edit;
+    edit.SetColumnFamilyID(cf_id);
+
+    std::vector<uint64_t> files;
+    Status s = it->GetBlobFilesInRanges(ranges, n, include_end, &files);
+    if (!s.ok()) return s;
+    for (auto file_number : files) {
+      edit->DeleteBlobFile(file_number);
+    }
+    s = LogAndApply(edit);
+    if (!s.ok()) return s;
+
+    ROCKS_LOG_INFO(db_options_.info_log,
+                   "%d blob files are deleted in the range [%s, %s%c",
```

Comment:
print all the files being deleted?

Reply:
try to test `auto-merge` bot, will add some logs in other PR later